swagger/swaggerDoc.yaml

```diff
@@ -0,0 +1,849 @@
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+          examples:
+            text/plain: "Empty Response"
+  /api/export_data:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+  /api/fbos_config:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    put:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+  /api/firmware_config:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    put:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+  /api/public_key:
+    get:
+      produces:
+      - "text/plain"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+            description: "Empty Response\n"
+  /api/tokens:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    post:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"user\": {\n    \"email\": \"nya@hoppe.io\",\n\
+            \    \"password\": \"password\"\n  }\n}"
+      responses:
+        200:
+          description: "Status 200"
+  /api/web_app_config:
+    get:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    put:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+  /app/{imageName}:
+    get:
+      description: "imageName=nope.jpg"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    parameters:
+    - name: "imageName"
+      in: "path"
+      required: true
+      type: "string"
+  /{jsonFile}:
+    post:
+      description: "jsonFile=csp_reports.json"
+      produces:
+      - "application/json"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+          examples:
+            application/json: "{\n  \n}"
+        400:
+          description: "Status 400"
+          schema:
+            type: "object"
+            example: "{\n  \"problem\": \"Crashed while parsing report\"\n}"
+    parameters:
+    - name: "jsonFile"
+      in: "path"
+      required: true
+      type: "string"
+  /tos_update:
+    get:
+      description: "???\nwhat is a tos_update??"
```

Comment:
"Terms of Service Update". Most self hosting users do not have Terms of Service for use of their servers, but some (ours) do. `tos_update` is an endpoint that handles user consent when the TOS changes.